src/rtc/rtc_s1.py

```diff
@@ -592,11 +593,36 @@ def run_parallel(cfg: RunConfig, logfile_path, flag_logger_full_format):
         list_burst_id = list(cfg.bursts.keys())
         for index_child, processing_result in \
                 enumerate(processing_result_list):
-            if processing_result != 0:
-                msg_failed_child_proc += (
-                    f'"{burst_runconfig_list[index_child]}"'
-                    ' for burst ID '
-                    f'"{list_burst_id[index_child]}"\n')
+
+            burst_id = list_burst_id[index_child]
+            output_dir_bursts = os.path.join(output_dir, burst_id)
+
+            # Conservative change to avoid breaking OPERA RTC-S1
+            # production. If there's at least one unsuccessful
+            # product, stop execution and delete ALL processed
+            # products to avoid sending half-empty products to the DAAC
+            # Do not delete log files
+            for ext in ['h5', 'tif', 'png']:
```

Comment:
```suggestion
            for ext in ['.h5', '.tif', '.png']:
```
I think it should be okay with the extension dots, but just to be sure.

Reply:
Thank you, @seongsujeong , I added the dot before using the variable `ext` in https://github.com/opera-adt/RTC/pull/85/commits/65053862e5a39a473bfa8491858922723dfb249e